atomic_reactor/constants.py

```diff
@@ -161,6 +162,8 @@
 
 REPO_CONTAINER_CONFIG = 'container.yaml'
 REPO_CONTENT_SETS_CONFIG = 'content_sets.yml'
+REPO_FETCH_ARTIFACTS_URL = 'fetch-artifacts-url.yaml'
```

Comment:
fetch artifacts plugin had originally

NVR_REQUESTS_FILENAME = 'fetch-artifacts-koji.yaml'
URL_REQUESTS_FILENAME = 'fetch-artifacts-url.yaml'

why are we renaming configuration files??? from 'fetch-artifacts-koji.yaml' to 'fetch-artifacts-nvr.yaml'

Reply:
@rcerven Good catch! Sorry for this mistake. It is fixed in latest updated commit.